managing-os/templates/archlinux.md

```diff
@@ -204,20 +210,13 @@ Main maintainer of this template is [Olivier Médoc](mailto:o_medoc@yahoo.fr).
 <br>
 <br>
 
-        *   Screen '**Get Resources**' wants to download additional packages needed for the choosen plugin/s.
+        *   Screen '**Get sources**' wants to download additional packages needed for the choosen plugin/s.
         
-            *   Select '**OK**' Prss '**Enter**'
-            
-        *   Upon completion you will get choose '**OK**' to proceed to the next screen
-        
-            *   Press '**Enter**'
+            *   Select '**No**' Press '**Enter**'
```

Comment:
> @marmarek: Ok to merge?

Mhm, :+1:

-- 
Best Regards,
Marek Marczykowski-Górecki
Invisible Things Lab
A: Because it messes up the order in which people normally read text.
Q: Why is top-posting such a bad thing?